File: C/Game.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int main()
{
    int number, guess, nattempt=1;
    srand(time(0));
    number = rand() % 100 + 1; 
    do
    {
        printf("Enter any Number:");
        scanf("%d", &guess);
        if (number < guess)
        {
            printf("Lower Number Please\n");
        }
        else if (number > guess)
        {
            printf("Greater Number Please\n");
        }
        else
        {
            printf("You Won Congratulations\nYou Guessed in %d attempt",nattempt);
        }
        nattempt++;
    } while (number != guess);
return 0;
}

File: pyhton/game.py
import random
number = random.randint(1, 100)
guess = None
nattempt = 1

while (guess != number):
        guess = int(input("Enter any Number: "))
        if guess < number:
            print("Greater Number Please")
        elif guess > number:
            print("Lower Number Please")
        else:
            print(f"You Won! Congratulations!\nYou guessed it in {nattempt} attempt(s).")
        nattempt += 1